_components/text-input/text-input.md

```diff
@@ -26,12 +26,12 @@ tags:
   - box
   - form controls
 variants:
-  - variant: "`usa-input--error`"
+  - variant: "`.usa-input--error`"
     description: The error state of the text input.
-  - variant: "`usa-input--success`"
+  - variant: "`.usa-input--success`"
     description: The success state of the text input.
-  - variant: "`usa-input--[width]`"
-    description: Displays an input at a specific width. Accepts `2xs` (4ex), `xs` (7ex), `sm` or `small` (10ex), `md` or `medium` (20ex), `lg` (30ex), `xl` (40ex), and `2xl` (50ex).
+  - variant: "`.usa-input--[width]`"
+    description: Displays an input at a specific width. Accepts `2xs` (5ex), `xs` (9ex), `sm` or `small` (13ex), `md` or `medium` (20ex), `lg` (30ex), `xl` (40ex), and `2xl` (50ex).
```

Comment:
Is this variant still used? I don't see any references to it in the design system and I'm unable to get it to change the width in Storybook 🤔

Reply:
Scratch that, I realized I had some conflicting styles!